service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationEventHandlerTestData.java

```diff
@@ -57,6 +58,15 @@ public static CallbackRequest appendSendToCtscOnCallback() {
             .build();
     }
 
+    public static Map<String, Object> getExpectedAllocatedJudgeNotificationParameters() {
```

Comment:
I would keep this in the `NotifyAllocatedJudgeEventHandlerTest` unless you think that other tests are going to be able to make use of it

Reply:
will do :)